frontend/src/features/workspace/WorkspacePage.tsx

```diff
@@ -32,6 +60,43 @@ export const WorkspacePage = (): JSX.Element => {
 
   const createFormModalDisclosure = useDisclosure()
 
+  // Whether to display the admin feedback based on session eligibility and time of prev feedback seen
+  useEffect(() => {
+    if (
+      // user details is loaded
+      !isLoading &&
+      // TODO: create mobile version of admin feedback
+      !isMobile &&
+      // admin session eligibility
+      isAdminFeedbackEligible &&
+      // if feedbackTime has not been seen
+      (!lastFeedbackTime ||
+        // or if last feedback time seen is more than frequency (frequency env var must be defined)
+        (!!adminFeedbackDisplayFrequency &&
+          currentTime - lastFeedbackTime > adminFeedbackDisplayFrequency))
```

Comment:
This `if` statement is getting long, might make sense to declare them as constants which can become self-documented code.

Reply:
[8bfd664](https://github.com/opengovsg/FormSG/pull/6465/commits/8bfd664ae1ecef25c445bd37908a001517c77eb6) Have abstracted out the admin eligibility portion from the broader check to display feedback!